lms/services/assignment.py

```diff
@@ -227,8 +227,7 @@ def get_assignments(
         :param course_ids: only return assignments that belong to this course.
         :param h_userids: return only assignments where these users are members.
         """
-
-        query = select(Assignment)
+        query = select(Assignment).where(Assignment.title.is_not(None))
```

Comment:
Not making this an option/parameter.

We only use this for the dashboards, we if we ever need the empty tiles one we can add something like `include|exclude_empty_titles`.